sample-app/config/routes.rb

```diff
@@ -1,9 +1,12 @@
 Rails.application.routes.draw do
+  get 'users/new'
   scope "(:locale)", locale: /en|vi/ do
     root "static_pages#home"
-
-    get "static_pages/home"
-    get "static_pages/about"
-    get "static_pages/help"
+    
+    get "static_pages/home", to: "static_pages#home"
+    get "static_pages/about", to: "static_pages#about"
+    get "static_pages/help", to: "static_pages#help"
+    get "static_pages/contact", to: "static_pages#contact"
+    get "users/signup", to: "users#new"
```

Comment:
giải thích cho c 
get "users/signup", to: "users#new"
là gì e nhỉ. để "users#create" có dc k e

Reply:
get "users/signup", to: "users#new" 
phần  get "users/signup" sẽ là phần hiện trên url ạ 
phần to: "users#new"  là đường dẫn thật sự đến view ạ 
Nếu để users#create sẽ không được ạ

Reply:
e tìm hiểu lại đoạn này rồi trả lời lại nhé
sao để `users#create` lại không được ?

Reply:
get "users/signup", to: "users#new"
dạ em đã tìm hiểu lại ạ. Theo em được biết thì đoạn code trên sẽ điều hướng get request có url là "users/signup" tới action new trong users controller  ạ. Và vì ở trong user controller của em hiện tại chưa có action create nên không thể để users#create đc ạ sẽ phát sinh lỗi ạ